.github/workflows/ci.yml

```
@@ -120,3 +120,60 @@ jobs:
         curl -L https://github.com/ndmitchell/hlint/releases/download/v${HLINT_VERSION}/hlint-${HLINT_VERSION}-x86_64-linux.tar.gz --output hlint.tar.gz
         tar -xvf hlint.tar.gz
         ./hlint-${HLINT_VERSION}/hlint src/ test/
+
+  stan:
+    name: stan
```

Can we add a GHC version to the name to quickly see what GHC is used?
```suggestion
    name: stan / ghc ${{ env.GHC_VERSION }}
```

---

Looks like `env` is not accessible on `jobs.<job_id>.name` 😢 

https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability

Error on my fork: https://github.com/german1608/iris/actions/runs/3202338434/workflow

For now, we could hardwire the GHC version?